extension-manifest-v2/src/classes/EventHandlers.js

```diff
@@ -331,6 +331,9 @@ class EventHandlers {
 		// process the tracker asynchronously
 		// very important to block request processing as little as necessary
 		setTimeout(() => {
+			if (tabInfo.getTabInfo(tab_id, 'url') !== eventMutable.tabUrl) {
```

Comment:
As I understand, this relies on the adblocker engine to handle the requests later?

For instance, beacon requests like this will be skipped:
```
"https://www.google.com/gen_204?atyp=i&bb=1&ei=vQotZb6tM4WE9u8P16-k2As&ct=slh&v=t1&im=M&m=HV&pv=0.765146168780363&me=1:1697450714401,x:0,V,0,0,704,1059:0,N,1,vQotZb6tM4WE9u8P16-k2As:0,R,1,101,24,36,92,33:0,R,1,CAgQAA,24,88,764,40:0,R,1,CAwQAQ,24,88,73,40:0,R,1,CAcQAQ,103,88,66,40:0,R,1,CAkQAQ,175,88,65,40:0,R,1,CAsQAQ,246,88,75,40:0,R,1,CEsQAQ,327,88,64,40:0,R,1,CEwQAQ,397,88,64,40:0,R,1,CE0QAQ,467,88,86,40:0,R,1,CAQQGA,28,269,652,1904:0,R,1,CBoQAA,28,269,600,117:0,R,1,CBwQAA,28,416,600,117:0,R,1,CA4QAA,28,577,652,150:0,R,1,CA8QAQ,28,614,652,48:0,R,1,CBMQAQ,28,663,652,48:0,R,1,CB0QAA,28,771,600,117:0,R,1,CC0QAA,28,917,600,117:483,h,1,CBoQAA,o:1,h,1,CAQQGA,o:65,B,2601:535,h,1,CBMQAQ,i:0,h,1,CA4QAA,i:0,h,1,CAQQGA,i:118,h,1,CBMQAQ,o:0,h,1,CA8QAQ,i:66,h,1,CA8QAQ,o:17,h,1,CA4QAA,o:33,h,1,CBwQAA,i:620,G,1,CBwQAA,164,58,1:0,c,192,474:0,G,1,CBwQAA,164,58:0,G,1,CAQQGA,164,205:11,e,C&zx=1697450716351&opi=89978449" 
```

Reply:
`_processBug` is reporting only, does not relate to the action of blocking

Reply:
and yes, the goal is to ignore the beacons on subsequent page load